SignalMessaging/Views/ContactTableViewCell.m

```diff
@@ -18,7 +18,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-const NSUInteger kContactTableViewCellAvatarSize = 40;
+const NSUInteger kContactTableViewCellAvatarSize = 48;
```

Comment:
Unify the avatar heights. This will unify the cell heights except in cases where dynamic text sizes are very large.